### ISSUE TITLE
Nordic-Baltic Grassland Vegetation Database (NBGVD)

### ISSUE THREAD

**Issue by gbif-portal** [gbif/data-mobilization#452]
## Nordic-Baltic Grassland Vegetation Database (NBGVD)

Dataset link: https://vcs.pensoft.net/article/119968/download/suppl/31/

Region: Nordic-Baltic

Taxon: plantae

Type: sampling event

Why is this important: As of March 2024, it included 12,694 relevés recorded between 1910 and 2023.

Priority: medium

License: Not open

Bibliographic reference: Skobel N, Kozub Ł, Dembicz I, Boch S, Bruun HH, Chusova O, Golub V, Helm A, Iakushenko D, Pawlikowski P, Zaniewski P, Dengler J (2024) Nordic-Baltic Grassland Vegetation Database (NBGVD) – current state and future prospects. Vegetation Classification and Survey 5: 75-84. https://doi.org/10.3897/VCS.119968

Comments: Hans Henrik Bruun is based next door to GBIFS and is a friendly known contact

Dataholders contact information: hhbruun@bio.ku.dk

Users contact info: dschigel

**Comment by CecSve** [comment]
Apparently there are several regional databases that may all be possible to mobilize as individual datasets: https://edgg.org/databases/Regional-databases and some are relevant for @spalp and @OlegBorodin77.

More detailed contact information can be found for each DB.

**Comment by kingenloff** [comment]
@spalp Any updates on this one, Salza?

**Comment by spalp** [comment]
No, I haven't worked on this issue at all, sorry. I wanted to accumulate examples and good experience in datasets with complex hierarchical design. I know some of the authors in person (Dembicz, Juergen), might try to talk to them during some of our conferences. However, I do not know in person the first author. If anyone feels like talking to Bruun, please refer to the following datasets as good examples:
